spec/factories/aid_applications.rb

```diff
@@ -10,6 +10,10 @@
     city { Faker::Address.city }
     zip_code { '94103' }
 
+    mailing_street_address { Faker::Address.street_address }
+    mailing_city { Faker::Address.city }
+    mailing_zip_code { '03226' }
```

Comment:
For the factory, this would also imply that the `allow_mailing_address` should also be true. We should set that in the factory too so that it's a realistic object.